contracts/SpokePool.sol

```diff
@@ -1330,19 +1304,9 @@ abstract contract SpokePool is
         // Send each relayer refund address the associated refundAmount for the L2 token address.
         // Note: Even if the L2 token is not enabled on this spoke pool, we should still refund relayers.
         uint256 length = refundAmounts.length;
-        for (uint256 i = 0; i < length; ) {
+        for (uint256 i = 0; i < length; i++) {
```

Comment:
Pre-increment _may_ be cheaper, but may also be optimised away by the compiler.

```suggestion
        for (uint256 i = 0; i < length; ++i) {
```

Reply:
[relayData.exclusivityDeadline >= block.timestamp && relayData.exclusiveRelayer != msg.sender](https://docs.soliditylang.org/en/v0.8.11/control-structures.html#revert)

I think they're equivalent?

Reply:
The above link is for reversion (also note 0.8.11 rather than 0.8.22/23).

In any case, there are still many blog posts claiming that pre-increment is a bit cheaper than post-incremement.
https://medium.com/@solidity101/100daysofsolidity-gas-optimization-techniques-in-solidity-9a3eb4b327c5
https://medium.com/coinmonks/gas-saving-techniques-for-smart-contract-development-5daa7da22370
https://betterprogramming.pub/solidity-gas-optimizations-and-tricks-2bcee0f9f1f2

I haven't verified this by simulating, and it probably seems like something the optimiser should detect and handle. But it also doesn't hurt to just use pre-increment, since there are no side-effects.